src/main/java/org/truffleruby/core/string/StringSupport.java

```diff
@@ -1641,4 +1646,302 @@ public static boolean isAsciiCodepoint(int value) {
     }
 
     //endregion
+    //region undump helpers
+
+    private static final byte[] FORCE_ENCODING_BYTES = ".force_encoding(\"".getBytes();
+    private static final byte[] HEXDIGIT = "0123456789abcdef0123456789ABCDEF".getBytes();
```

Comment:
It should be `.getBytes(StandardCharsets.US_ASCII)` here, to not depend on the default Charset.

Reply:
Had to use `RopeOperations.encodeAsciiBytes` instead, because of `checkstyle`.